From: Cyril Brulebois <cyril@debamax.com>
Subject: Disable source maps.
Date: Wed, 21 May 2025 00:39:38 +0200

Without this, the path to the git checkout leaks in the static library
and in the Python module:
 - /usr/lib/x86_64-linux-gnu/libfrida-core-1.0.a
 - /usr/lib/python3/dist-packages/frida/_frida.abi3.so

Link: https://github.com/PiRogueToolSuite/deb-frida/issues/3

--- a/subprojects/frida-core/src/barebone/package.json
+++ b/subprojects/frida-core/src/barebone/package.json
@@ -4,7 +4,7 @@
   "description": "Script runtime used by Frida's Barebone backend",
   "type": "module",
   "scripts": {
-    "build": "frida-compile script-runtime/entrypoint.ts -o script-runtime.js -c"
+    "build": "frida-compile -S script-runtime/entrypoint.ts -o script-runtime.js -c"
   },
   "devDependencies": {
     "@types/frida-gum": "^19.0.0",
